hooks/000-provide-uids-gids.chroot

```diff
@@ -225,17 +227,17 @@ kvm:x:112:
 syslog:x:114:
 pkcs11:x:115:
 tss:x:116:
-input:x:107:
 render:x:117:
-sgx:x:119:
 _ssh:x:118:
+sgx:x:119:
```

Comment:
I don't think it is worth making this change. The current ordering is inherited from upstream core22:

https://github.com/snapcore/core-base/blob/9c9aa431346cc65eff3fa2484cf5fef5ffb208a0/hooks/000-provide-uids-gids.chroot#L213-L216

Changing the order just increases the chance of future merge conflicts.

Reply:
You are right. I'll revert that.

Reply:
Done.